spec/features/avo/select_field_spec.rb

```diff
@@ -229,4 +229,33 @@ def test_array
       end
     end
   end
+  describe "when options are multiple select" do
+    context "new" do
+      it "allow selection of multiple values" do
+        Avo::Resources::Product.with_temporary_items do
+          field :sizes, as: :select, multiple: true, options: { 'Large': :large, 'Medium': :medium, 'Small': :small}
+        end
```

Comment:
```suggestion
```

No need for `Avo::Resources::Product.with_temporary_items` since the `sizes` field is already configured on the dummy's resource.